### PR TITLE
GREEN-7000: Documentation for Add/Remove Job Permissions

### PR DESCRIPTION
**Pull request by jaydeesimon**

### CODE REVIEW

File: source/includes/harvest/_users.md [hunk 1]
```diff
@@ -76,12 +76,12 @@ id | The ID of the user whose future job permissions to retrieve
 
 An array of objects of the following:
 
-Parameter | Description
```
Comment (jaydeesimon) [inline source/includes/harvest/_users.md]: Unrelated add/remove job permissions. I added a Type column to keep it consistent.

File: source/includes/harvest/_users.md [hunk 2]
```diff
@@ -143,11 +143,11 @@ id | The ID of the user whose job permissions to retrieve
...
```
Comment (jaydeesimon) [inline source/includes/harvest/_users.md]: Unrelated add/remove job permissions. I added a Type column to keep it consistent.

File: source/includes/harvest/_users.md [hunk 3]
```diff
@@ -143,11 +143,11 @@ id | The ID of the user whose job permissions to retrieve
...
---------- | -----------
-id | The ID representing the job permission
-job_id | The ID of the job for the job permission
-user_role_id | The ID of the user role for the job permission
+Parameter | Type | Description
+--------- | ----------- | ------------
+id | integer | The ID representing the job permission
+job_id | integer | The ID of the job for the job permission
+user\_role\_id | integer | The ID of the user role for the job permission
```
Comment (mmurph24) [inline source/includes/harvest/_users.md]: This feels a little confusing to me. What if it was something like 

+job_id | integer | The ID of the job to be added or removed
+user\_role\_id | integer | The ID of the user role to be assigned for this job

File: source/includes/harvest/_users.md [hunk 4]
```diff
@@ -353,3 +353,85 @@ email | Yes | string | The user's email address. Must be a valid email address.
...
+-d '{ "job_id": {job_id}, "user_role_id": {user_role_id} }'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```
+{
+  "id": 271232,
```
Comment (mmurph24) [inline source/includes/harvest/_users.md]: Will the user ever specify the ID here? Won't they just choose the job_id and user_role_id?

File: source/includes/harvest/_users.md [hunk 5]
```diff
@@ -143,11 +143,11 @@ id | The ID of the user whose job permissions to retrieve
...
 An array of objects of the following:
 
-Parameter | Description
---------- | -----------
-id | The ID representing the job permission
-job_id | The ID of the job for the job permission
-user_role_id | The ID of the user role for the job permission
+Parameter | Type | Description
+--------- | ----------- | ------------
+id | integer | The ID representing the job permission
```
Comment (mmurph24) [inline source/includes/harvest/_users.md]: What about "The unique ID associating a specific user to a job and user role."

File: source/includes/harvest/_users.md [hunk 6]
```diff
@@ -353,3 +353,85 @@ email | Yes | string | The user's email address. Must be a valid email address.
 send_email_invite* | No | boolean | If true, an email will be sent to the above email address inviting them to login. If false, nothing happens. Default is false.
```
Comment (mmurph24) [inline source/includes/harvest/_users.md]: Small change 

> If true, an email will be sent to the user alerting them of any new job permissions that have been assigned to them. If false, nothing happens. Default is false.

File: source/includes/harvest/_users.md [hunk 7]
```diff
@@ -76,12 +76,12 @@ id | The ID of the user whose future job permissions to retrieve
...
 
-Parameter | Description
---------- | -----------
-id | The ID representing the future job permissions
-office_id | The ID of the office. The user role will be granted when a job is created belonging to this office.
-department_id | The ID of the department. The user role will be granted when a job is created belonging to this department.
-user\_role\_id | The ID of the user role that will be granted to the user.
+Parameter | Type | Description
+--------- | ----------- | ---------------
+id | integer | The ID representing the future job permissions
```
Comment (mmurph24) [inline source/includes/harvest/_users.md]: What about "The unique ID associating a specific user to a future job permission."

File: source/includes/harvest/_users.md (hunk 4 above)
Comment (jaydeesimon) [inline source/includes/harvest/_users.md]: Yes. That's actually supposed to say, `The above command returns a JSON response, structured like this:`

**Pull request merged**